writer_test.go

```diff
@@ -365,7 +365,7 @@ func TestEncodeMediaPlaylistWithDefaultMap(t *testing.T) {
 	if e != nil {
 		t.Errorf("Set map to segment failed: %s", e)
 	}
-
+	//fmt.Println(p.Encode().String())
```

Comment:
Debug info?

Reply:
yeah, saw other tests same way, so assumed that was a pattern for other people coming in to get started with tests easily...